bitcoin.cpp

```diff
@@ -27,6 +29,12 @@ class CNode
     int64 doneAfter;
     CAddress you;
 
+public:
+    uint64_t maxGrapheneVersion = 0;
+    uint64_t electrsVersion = 0;
+    uint64_t capdVersion = 0;
```

Comment:
since all others info we gathered are private I wonder why declaring those members as public? Any specific need?

Reply:
I mean for these variables to be part of the external interface to this class.  For example, if you want to know the CNode's electrsVersion, read the variable.

Reply:
I see. 

Would it make sense for all the other variables that now are read via accessors e.g. `GetClientVersion()`, `GetClientSubVersion()`, ... ?